docker_test.go

```diff
@@ -2,11 +2,15 @@ package dockertest_test
 
 import (
 	"database/sql"
-	. "github.com/ory-am/dockertest"
-	"github.com/stretchr/testify/require"
-	"gopkg.in/mgo.v2"
+	"strings"
 	"testing"
 	"time"
+
+	"gopkg.in/mgo.v2"
+
+	"github.com/mattbaird/elastigo/lib"
+	. "github.com/ninnemana/dockertest"
```

Comment:
This has to be `"github.com/ory-am/dockertest"` not `"github.com/ninnemana/dockertest"`